src/Rule/Trait/BeforeValidationTrait.php

```diff
@@ -19,6 +19,9 @@ public function getSkipOnEmpty(): bool
         return $this->skipOnEmpty;
     }
 
+    /**
+     * @psalm-suppress InvalidNullableReturnType NEED FIX BEFORE RELEASE!!!
```

Comment:
?

Reply:
Fix this psalm error required modify many files. I'll make this in separate PR.